### DISCUSSION TITLE
[@types/react-dom] unmatch dependency version

### DISCUSSION THREAD

**Discussion by jiyeol-lee** [DefinitelyTyped/DefinitelyTyped#59801] [Issues with a @types package] [answered]
Hello, I have a problem using @types/react-dom now because, in node_modules, they use an old version of @types/react. For example, in @types/react, you decided to remove {} in ReactFragment but in @types/react-dom, it's not removed. My code is complaining because of this. Can you guys check this?

![Screen Shot 2022-04-10 at 8 08 10 AM](https://user-images.githubusercontent.com/41712713/162617427-f65bd9a9-b880-410e-861e-9bd54c77da78.png)

[a comment by an automated account is not shown]

**Comment by eps1lon** [comment, marked as answer]
You need to tell your package manager to only install a single version of `@types/react`.

For Yarn you can run `npx yarn-deduplicate --packages @types/react`.

I don't know how to do that for npm beyond removing the existing `package-lock.json` and running `npm install` again.

**Reply by jiyeol-lee** [reply]
Thank you so much! It works like a magic!
